HLTrigger/Configuration/python/Phase2/trackerLocalReco_cff.py

```diff
@@ -0,0 +1,84 @@
+import FWCore.ParameterSet.Config as cms
+
+siPhase2Clusters = cms.EDProducer(
+    "Phase2TrackerClusterizer",
+    maxClusterSize=cms.uint32(0),
+    maxNumberClusters=cms.uint32(0),
+    src=cms.InputTag("mix", "Tracker"),
+)
+
+siPixelClusters = cms.EDProducer(
+    "SiPixelClusterProducer",
+    ChannelThreshold=cms.int32(1000),
+    ClusterMode=cms.string("PixelThresholdClusterizer"),
+    ClusterThreshold=cms.int32(4000),
+    ClusterThreshold_L1=cms.int32(4000),
+    ElectronPerADCGain=cms.double(600.0),
+    MissCalibrate=cms.bool(False),
+    Phase2Calibration=cms.bool(True),
+    Phase2DigiBaseline=cms.double(1200),
+    Phase2KinkADC=cms.int32(8),
+    Phase2ReadoutMode=cms.int32(-1),
+    SeedThreshold=cms.int32(1000),
+    SplitClusters=cms.bool(False),
+    VCaltoElectronGain=cms.int32(1),
+    VCaltoElectronGain_L1=cms.int32(1),
+    VCaltoElectronOffset=cms.int32(0),
+    VCaltoElectronOffset_L1=cms.int32(0),
+    maxNumberOfClusters=cms.int32(-1),
+    mightGet=cms.optional.untracked.vstring,
+    payloadType=cms.string("Offline"),
+    src=cms.InputTag("simSiPixelDigis", "Pixel"),
+)
+
+siPixelClusterShapeCache = cms.EDProducer(
+    "SiPixelClusterShapeCacheProducer",
+    mightGet=cms.optional.untracked.vstring,
+    onDemand=cms.bool(False),
+    src=cms.InputTag("siPixelClusters"),
+)
+
+siPixelRecHits = cms.EDProducer(
```

Comment:
from earlier comments from @fwyzard 
I understood that these will be renamed to hltSiPixelRecHits.

Reply:
Renamed to hltPhase2SiPixelRecHits